blockchain/types/accountkey/account_key_weighted_multi_sig.go

```diff
@@ -75,7 +77,19 @@ func (a *AccountKeyWeightedMultiSig) Equal(b AccountKey) bool {
 		a.Keys.Equal(tb.Keys)
 }
 
-func (a *AccountKeyWeightedMultiSig) Validate(r RoleType, recoveredKeys []*ecdsa.PublicKey, from common.Address) bool {
+// Validate returns validation result (bool) and valid&unique signature number (int)
+func (a *AccountKeyWeightedMultiSig) Validate(r RoleType, recoveredKeys []*ecdsa.PublicKey, from common.Address, currentBlockNumber uint64) (bool, int) {
```

Comment:
I'm sorry I couldn't give you a good review.
I have a question...
The original function validates the input and returns validation result.
Now, this function also returns validSigNum. Is it okay to take the naming as same?
When looking at the code in which this function is used, I think it is difficult to infer returned int value.

Reply:
Sorry never mind.